components/Day.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { View, Text, StyleSheet, Image } from 'react-native';
+import PropTypes from 'prop-types';
+
+const styles = StyleSheet.create({
+  icon: {
+    height: 55,
+    width: 55,
+  },
+  floatRight: {
+    alignSelf: 'flex-end',
+  },
+  floatleft: {
+    alignSelf: 'flex-start',
+  },
+  baseText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 18,
+    textAlign: 'center',
+  },
+  smallText: {
+    fontFamily: 'Roboto',
+    color: '#fff',
+    fontSize: 14,
+    textAlign: 'center',
+  },
+});
+
+const Day = (props) => {
```

Comment:
You can do it like that
```suggestion
const Day = ({ weather, style }) => {
```